parser_web.py

```diff
@@ -0,0 +1,39 @@
+import requests
+from bs4 import BeautifulSoup
+import csv
```

Comment:
Could you create the csv fil with `pands` instead of `csv`?

Reply:
Could you please add the package `beautifulsoup4` in the list of dependencies in the file `binder/environment.yml`?